package moe.kageru.kagebot

import moe.kageru.kagebot.Log.log
import java.lang.System

fun main() {
    //try {
        Kagebot()
    /*} catch (e: Exception) {
        log.warning("An exception occurred in the main thread, exiting.\n${e.stackTrace.joinToString("\n")}")
        System.exit(1)
    }*/
}
